int32 boat_id
geometry_msgs/Pose2D center #中心座標と角度
geometry_msgs/Pose2D[] corners #コーナーの座標
